### PR TITLE
Coding Practice 22

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/codingPractice/Flight.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    public int getConfirmationNumber() {
+        return confirmationNumber;
+    }
+
+    public void setConfirmationNumber(int confirmationNumber) {
+        this.confirmationNumber = confirmationNumber;
+    }
+
```
Comment (anushreeSonak) [inline simple/src/codingPractice/Flight.java:61]: remove extra space

File: simple/src/codingPractice/Main.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        app.bookFlight(12345670, "Martin Nadine", "New York", "London",
+                "2022-08-01", 1, 700.00);
+        app.bookFlight(67843513, "Jennifer Ulrike", "New York", "London",
+                "2022-08-01", 1, 655.00);
+        app.bookHotel(98765432, "Martin Nadine", "London", "2022-09-01",
+                "2022-09-05", 1, 100.00);
+        // Cancel a flight or hotel reservation based on reservation number
+        app.cancelReservation(12345670);
+
+
```
Comment (anushreeSonak) [inline simple/src/codingPractice/Main.java:19]: remove above extra spaces

File: simple/src/codingPractice/Main.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package codingPractice;
+
+public class Main {
```
Comment (anushreeSonak) [inline simple/src/codingPractice/Main.java:3]: Class Name should be meaningful.

**Review by anushreeSonak — APPROVED** [review]

File: simple/src/codingPractice/TravelApp.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
+package codingPractice;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class TravelApp {
+    private ArrayList<Flight> flights;
```
Comment (prashantkawa) [inline simple/src/codingPractice/TravelApp.java:7]: add space here

File: simple/src/codingPractice/TravelApp.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    public void bookHotel(int hotelId, String guestName, String location, String checkIn, String checkOut,
+                          int numGuests, double price) {
+        Hotel hotel = new Hotel(hotelId, guestName, location, checkIn, checkOut, numGuests, price);
+        int confirmationNumber = generateConfirmationNumber();
+        hotel.setConfirmationNumber(confirmationNumber);
+        this.hotels.add(hotel);
+        System.out.println("Hotel booked! Confirmation number: " + confirmationNumber);
+    }
+
+    public void cancelReservation(int confirmationNumber) {
```
Comment (prashantkawa) [inline simple/src/codingPractice/TravelApp.java:43]: Removing flight details or cancelling reservation here?

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: simple/src/codingPractice/TravelApp.java (hunk 4 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/TravelApp.java:7]: Done

File: simple/src/codingPractice/TravelApp.java (hunk 5 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/TravelApp.java:44]: Removing flight details

File: simple/src/codingPractice/Main.java (hunk 3 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/Main.java]: done

File: simple/src/codingPractice/Main.java (hunk 2 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/Main.java]: done

File: simple/src/codingPractice/Flight.java (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/Flight.java]: done

**Review by prashantkawa — APPROVED** [review]

**Pull request merged**